user/src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './Footer.scss'
+import { contacts } from './data.js'
+import ContactBlock from '../ui/contact-block/ContactBlock';
+
+const Footer = () => {
+    return (
+        <footer className='footer'>
+            <div className='container'>
+                <div className='footer-wrapper'>
+                    <div className='footer-top'>
+                        <div className='footer__title'>
+                            <span className='text__title text__title--bold me-2'>FOLLOW US</span>
+                            <span className='text__title'>TO GET A DISCOUNT</span>
+                        </div>
+                        <div className='footer__contact'>
+                            <ContactBlock data={contacts} />
+                        </div>
+                    </div>
+                    <div className='footer-bot'>
+                        <div className='footer__text'>
+                            © Copyrights CodexThemes
```

Comment:
```suggestion
                            &copy; Copyrights CodexThemes
```
Những kí tự đặc biệt như này, hoặc các kí tự liên quan tới code như & < > ' " thì encode nó nhé